iso15118/shared/states.py

```diff
@@ -264,6 +269,7 @@ def create_next_message(
             except ValidationError as exc:
                 logger.exception(exc)
                 raise exc
+            self.next_msg = to_be_exi_encoded
```

Comment:
This `next_msg` attribute of the State class is a bit useless and I think has even the wrong name as the purpose was to save the last message sent, I believe...for purposes of debugging, but we are not using it for that purpose. So, at least I would change the name...what do you think?

Also this `self.next_msg` and `next_msg` just makes things confusing...

Reply:
Yes, I felt the same. Initially I thought we could remove next_msg completely but then realised it is being used in tests (and will be of use in the future) and also in comm_session as follows just before V2GTP packet is sent:
```
         Args:
            message: A V2GTPMessage
        """
        logger.info(f"Sending {str(self.current_state.next_msg)}")
```
I felt it a bit odd to read the above as "sending next message from current state", rather than reading "sending message from current state" as this is the message output from "current_state"

I've now renamed the field to "message". Do you think that reads a bit better?

Also came across a bug where the above `logger.info` was printing `Sending None` with the new changes.
So have added a Base64 "wrapper" to hold the base64 CertificateInstallationRes to help with this.